doc/bitcoin-conf.md

```diff
@@ -30,6 +30,28 @@ Network specific options can be:
 - placed into sections with headers `[main]` (not `[mainnet]`), `[test]` (not `[testnet]`) or `[regtest]`;
 - prefixed with a chain name; e.g., `regtest.maxmempool=100`.
 
+Options that are either prefixed with a chain name or placed
+into sections with headers take precedence over options
+without network specific information. If multiple values
+for the same option is found with the same precedence, the 
+first one is generally chosen. 
+
+This means that given the following configuration, `rpcport` is set to `3000`:
+
+```
+regtest=1
+rpcport=2000
+regtest.rpcport=3000
+
+[regtest]
+rpcport=4000
+```
+
+Note that there's no way to end a network section without 
```

Comment:
In commit "Add note on precedence of options in bitcoin.conf" (f6ba37508663237c6f9d72d2d71a0307cbc6b78d)

Would drop this paragraph. I think it's pointing out trivia that's not useful (and also kind of obvious if you think about it). I'm pretty sure no one has any need or desire to end network sections. The only reason I brought up ending network sections earlier is to explain why I couldn't write an example of network setting in a section taking precedence over a network setting with a prefix.